tests/publiccloud/aistack_rbac_run.pm

```diff
@@ -0,0 +1,70 @@
+use Mojo::Base 'publiccloud::basetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::utils;
+use version_utils;
+
+sub run_python_script {
+    my $script = shift;
+    my $logfile = "output.txt";
+    record_info($script, "Running python script: $script");
+    assert_script_run("$script");
+    assert_script_run("./$script 2>&1 | tee $logfile");
+}
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $instance = $self->{my_instance} = $args->{my_instance};
+    my $provider = $self->{provider} = $args->{my_provider};
```

Comment:
```suggestion
    my $instance = $args->{my_instance};
    my $provider = $args->{my_provider};
```

Reply:
aren't the `$self` parameter needed too, for passing those ones to other possible modules?